src/tests-rosetta/api.ts

```diff
@@ -129,6 +130,30 @@ describe('Rosetta API', () => {
 
     const block = blockData.block, genesisBlock = genesisData.block;
 
+    nock('http://127.0.0.1:20443')
+      .get('/v2/neighbors')
+      .reply(200, {
+        sample: [],
+        inbound: [],
+        outbound: []
+      });
+    nock('http://127.0.0.1:20443')
+      .get('/v2/info')
+      .reply(200, {
+        burn_block_height: block.burn_block_height,
+        burn_consensus: block.burn_block_hash,
+        exit_at_block_height: null,
+        network_id: 1,
+        parent_network_id: 1,
+        peer_version: 1,
+        server_version: 1,
+        stable_burn_block_height: block.burn_block_height,
+        stable_burn_consensus: block.burn_block_hash,
+        stacks_tip: block.block_hash,
+        stacks_tip_burn_block: block.burn_block_height,
+        stacks_tip_height: block.block_height,
+        unanchored_tip: ''
+      });
```

Comment:
This and a few other test fixes look familiar -- similar changes were required in the `stacks-2.1` branch:
* https://github.com/hirosystems/stacks-blockchain-api/pull/1280/files#diff-352542c23d3fe38ede85cc2611da70e48ef527e1b09babb12bc6bb4ad263f2e0
* https://github.com/hirosystems/stacks-blockchain-api/pull/1279/files

It looks like some of the issues were caused by the bump from only `stacks-blockchain:2.05.0.0.0` (as opposed to `stacks-blockchain:next / 2.1`).

In the API's `stacks-2.1` branch, a larger refactor was done to fix these. Stubbing out these responses could potentially hide bugs if the stacks-node behavior diverges from these stubbed behaviors. However, we can probably address that later once we have to merge the `stacks-2.1` branch. In the meantime, seems okay to merge.

Reply:
Thanks for the note, I'll keep that in mind.

I did this mock only to skip the stacks node dependency on this test. Integration tests were already done in `construction.ts` so and adding a new event server here created problems given that the stacks node block height did not restart at 1 (it stayed at the previous height created by other tests).